ui/projects/DeployByEnvironment.qml

```diff
@@ -0,0 +1,58 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Item {
+    id: root
+
+    property var environment;
+
+    width: parent.width;
+    height: parent.height;
+
+    Rectangle {
+        id: rect;
+
+        color: "lightgray";
+        width: parent.width;
+        height: parent.height;
+
+        Text {
+            id: env
+            padding: 50
+            text: root.environment.name
+            font.pixelSize: 20
+            font.family: "AvantGarde-Medium"
```

Comment:
FYI: will probably not be available for everyone / every platforms. Should not be an issue anyway but to fix in another release.

Reply:
I just removed font.family, not really useful